### ISSUE TITLE
Create a second connector proxy for python libraries

### ISSUE THREAD

**Issue by harmeet-status** [sartography/spiff-arena#652]

**Comment by danfunk** [comment]
As each connector is an independent project, you can create whatever dependencies you need. We don't currently support multiple connector proxys, and that would be a valuable ticket - as there is some risk of conflicting libraries as we add a large number connectors. 

Moving to low priority, as the specific issues raised here are no longer a problem.

**Comment by harmeet-status** [comment]
As the connector proxy is a seperate repo to the Spiff repo, can we just start to use the libraries that we want and deal with your comment above @danfunk at a later stage? We only thinking about introducing 1 library atm.

**Comment by burnettk** [comment]
Yeah, there is no need to have a second connector proxy and there is nothing preventing the addition of new libraries to the existing connector.

**Comment by harmeet-status** [comment]
@MarD0607 wants to push a OCR library to the connector-proxy repo

**Comment by MarD0607** [comment]
Yes, actually create a new connector for a python library, just like we have the smtp connector

**Comment by burnettk** [comment]
sure, creating a new connector seems good 👍

**Comment by MarD0607** [comment]
@burnettk I have only one problem:
- after uploading the file, in the task data I can find the document id and name
- I need to get the document from the location where it is stored in Spiff and give it to the connector for processing

**Comment by burnettk** [comment]
Project no longer active.
